<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditorModuleContent</class>
 <widget class="QWidget" name="MapEditorModuleContent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>457</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>61</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>61</height>
      </size>
     </property>
     <widget class="QCheckBox" name="m_pShowGrid">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="m_pRevert">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&lt;-</string>
      </property>
      <property name="shortcut">
       <string/>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="autoRepeatDelay">
       <number>250</number>
      </property>
     </widget>
     <widget class="QFrame" name="m_pLayer">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QSpinBox" name="m_pCurLayer">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="m_pMapTreeLabel_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>layer no.:</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="m_pLayerBackground">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>background</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="m_pLayerForeground">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>foreground</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="m_pMapTreeLabel_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>grid:</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_pMapTreeLabel_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>zoom:</string>
      </property>
     </widget>
     <widget class="QSlider" name="m_pZoom">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="sliderPosition">
       <number>100</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="m_pMapTabs">
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
